cmd/fortifyExecuteScan.go

```diff
@@ -1141,8 +1141,7 @@ func determinePullRequestMerge(config fortifyExecuteScanOptions) (string, string
 func determinePullRequestMergeGithub(ctx context.Context, config fortifyExecuteScanOptions, pullRequestServiceInstance pullRequestService) (string, string, error) {
 	number := "0"
 	author := ""
-	options := github.PullRequestListOptions{State: "closed", Sort: "updated", Direction: "desc"}
-	prList, _, err := pullRequestServiceInstance.ListPullRequestsWithCommit(ctx, config.Owner, config.Repository, config.CommitID, &options)
```

Comment:
The attributes offered by `PullRequestListOptions` are not supported by the Github API, thus it got replaced by `ListOptions` in one of the recent releases of go-github.